pywinauto/unittests/test_application.py

```diff
@@ -434,12 +434,12 @@ def _cabinetwclass_exist():
 
     def test_wait_cpu_usage_lower_uia(self):
         """Test that wait_cpu_usage_lower() works correctly for UIA"""
-        app = Application(backend='uia')
-        app.start('notepad.exe')
+        self.app = Application(backend='uia')
+        self.app.start('notepad.exe')

```

Comment:
It adds a Codacy warning. But it's minor. Will try to revert back and see if this check is red or not.